unime/src/i18n/en/index.ts

```diff
@@ -165,6 +165,10 @@ const en = {
       TITLE: 'Support',
       ABOUT: {
         TITLE: 'About UniMe',
+        NAVBAR_TITLE: 'About UniMe',
+        VERSION: '0.4.9',
```

Comment:
`VERSION`, `TITLE_2` do not need to be translated as they're purely technical terms.

Reply:
You could have left `TITLE_1` in there, we can translate "Built with Tauri". I would name that key to `BUILT_WITH` or something, since it's not a title.